cli/server_test.go

```diff
@@ -973,26 +973,20 @@ func TestServer(t *testing.T) {
 
 			scanner := bufio.NewScanner(res.Body)
 			hasActiveUsers := false
-			hasWorkspaces := false
 			for scanner.Scan() {
 				// This metric is manually registered to be tracked in the server. That's
 				// why we test it's tracked here.
 				if strings.HasPrefix(scanner.Text(), "coderd_api_active_users_duration_hour") {
 					hasActiveUsers = true
 					continue
 				}
-				if strings.HasPrefix(scanner.Text(), "coderd_api_workspace_latest_build_total") {
-					hasWorkspaces = true
-					continue
-				}
```

Comment:
Side-effect of clearing the gauge when no db rows are loaded.
This wasn't strictly necessary for the test since other manually-registered metrics are included to validate this test.